tools/cli/main.js

```diff
@@ -16,6 +16,7 @@ var catalog = require('../packaging/catalog/catalog.js');
 var buildmessage = require('../utils/buildmessage.js');
 var httpHelpers = require('../utils/http-helpers.js');
 const archinfo = require('../utils/archinfo.js');
+import { isEmacs } from "../utils/utils.js";
```

Comment:
We better add a `isVi` function in `utils.js` just to be safe ... 🔥😱

Reply:
For the record, the speed is almost/exactly as fast as a native terminal session when running `vi`/`vim` split-screen with a terminal in `tmux` and `screen`. 😉